### DISCUSSION TITLE
Followed all of the steps to install wiki.js but getting error on last step.

### DISCUSSION THREAD

**Discussion by Enukora** [requarks/wiki#2850] [Help / Questions]
I followed all of the steps outlined for the installation instructions for wiki.js but when I try to perform the final step of running Wiki.js using the command "node server" I am getting an error saying 'Module_Not_Found.  Can someone assist me with what I may be doing wrong here?

<img width="597" alt="Screen Shot 2020-12-21 at 7 27 34 PM" src="https://user-images.githubusercontent.com/76460478/102841646-98ba6180-43c2-11eb-8269-103da07fd291.png">

Thank you

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

**Comment by Enukora** [comment]
I am using MacOS, latest version of Wiki.js, Database engine: PostgreSQL on Google Cloud Platform (GCP)
<img width="597" alt="Screen Shot 2020-12-21 at 7 27 34 PM" src="https://user-images.githubusercontent.com/76460478/102841646-98ba6180-43c2-11eb-8269-103da07fd291.png">

**Comment by NGPixel** [comment]
Which version of node are you using?

**Comment by Enukora** [comment]
I downloaded version 15.5.0

<img width="530" alt="Screen Shot 2020-12-22 at 5 06 28 PM" src="https://user-images.githubusercontent.com/76460478/102944446-0b394900-4478-11eb-9d38-074ee0a5f35c.png">

**Comment by NGPixel** [comment]
You need to run `node server` from the `wiki-js` folder.

**Reply by Enukora** [reply]
Finally I am able to run node server but now I am getting Database Connection Error:  

<img width="1444" alt="Screen Shot 2020-12-23 at 4 35 18 PM" src="https://user-images.githubusercontent.com/76460478/103044186-dc38db00-453c-11eb-85b7-f1c954b2773a.png">

Could this have something to do with the latest version of Node that I am running? I noticed that other people have had this error populate after they updated to the latest version but once they reverted back to a previous version then the issue was resolved. What are your thoughts?
